docs/API.md

```diff
@@ -77,6 +77,7 @@
     See [Extreme mode explained](extreme.md) for more detail.
   * `enabled` (boolean): enables logging. Default: `true`
   * `browser` (Object): browser only, may have `asObject` and `write` keys, see [Pino in the Browser](../readme.md#browser)
+  * `crlf` (boolean): emit `\r\n` instead of `\n`. Default: `false`.
```

Comment:
instead of "emit" can we say:

logs newline delimited JSON with `\r\n` instead of `\n`

Reply:
Changed, PTAL